### PR TITLE
Run CI for Bionic and Focal

### PR DESCRIPTION
**Pull request by jacobperron**
Replacing Trusty, which is EOL.

### CODE REVIEW

**Comment by jacobperron** [comment]
CI is green now :slightly_smiling_face:

**Review by ivanpauno — APPROVED** [review]

**Review by esteve — APPROVED** [review]

**Comment by esteve** [comment]
@jacobperron thanks! Perhaps the next step would be to migrate to GitHub actions. @jacobperron @ivanpauno what do you think?

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,11 @@
 language: python
 python: "3.5"
-dist: trusty
+jobs:
+  include:
+    - os: linux
+      dist: bionic
+    - os: linux
+      dist: focal
```
[the remark this answers is not in the document]
Comment (jacobperron) [reply .travis.yml:8]: Good point. The only dependency that has potential to differ between distributions is `enchant, coming from Ubuntu. I noticed that other colcon repositories have changed how they do the spellcheck test, such that they don't depend on `enchant`. We could make a similar change in a follow-up PR. Until then, I'm inclined to leave both Bionic and Focal :man_shrugging:

File: colcon_gradle/package_identification/gradle.py [hunk 2]
```diff
@@ -43,18 +42,17 @@ def identify(self, metadata):  # noqa: D102
...
     :param Path build_gradle: The path of the build.gradle file
     :rtype: dict
     """
-    # Content for dependencies
-    content_build_gradle = extract_content(build_gradle)
-    
     # Content for name
-    content_settings_gradle = extract_content(build_gradle.parent, 'settings.gradle')
+    content_settings_gradle = extract_content(
+        build_gradle.parent, 'settings.gradle')
```
[the remark this answers is not in the document]
Comment (jacobperron) [reply colcon_gradle/package_identification/gradle.py:55]: I think most colcon repos use the default line length of 79 characters. I'm indifferent if we choose to update the line-length for this repo though.

**Comment by jacobperron** [comment]
Switching to GitHub actions sounds okay to me. I just wanted to unblock CI and tried to make a minimum change here, but I'm happy to review a PR that switches us to GitHub actions.

**Pull request merged**